adhocteam/Head-Start-TTADP
# [TTAHUB-206] alphabetize multi selects

## Description of change
Multi selects alphabetize input as it is entered.
Multi select data is also sorted alphabetically on the approved report view, the review/draft report view, and the CSV exports.

## How to test
Create a new report. Enter multi select data in non-alpha order and see how it is displayed. View prior activity reports and note that all the multi-select fields have been alphabetized.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-206


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
